src/Faker/Resources/Company.it-IT.resx

```diff
@@ -0,0 +1,141 @@
+  </data>
+  <data name="Buzzwords2" xml:space="preserve">
+    <value>24 hour;24/7;3rd generation;4th generation;5th generation;6th generation;actuating;analyzing;assymetric;asynchronous;attitude-oriented;background;bandwidth-monitored;bi-directional;bifurcated;bottom-line;clear-thinking;client-driven;client-server;coherent;cohesive;composite;context-sensitive;contextually-based;content-based;dedicated;demand-driven;didactic;directional;discrete;disintermediate;dynamic;eco-centric;empowering;encompassing;even-keeled;executive;explicit;exuding;fault-tolerant;foreground;fresh-thinking;full-range;global;grid-enabled;heuristic;high-level;holistic;homogeneous;human-resource;hybrid;impactful;incremental;intangible;interactive;intermediate;leading edge;local;logistical;maximized;methodical;mission-critical;mobile;modular;motivating;multimedia;multi-state;multi-tasking;national;needs-based;neutral;next generation;non-volatile;object-oriented;optimal;optimizing;radical;real-time;reciprocal;regional;responsive;scalable;secondary;solution-oriented;stable;static;systematic;systemic;system-worthy;tangible;tertiary;transitional;uniform;upward-trending;user-facing;value-added;web-enabled;well-modulated;zero administration;zero defect;zero tolerance</value>
+  </data>
+  <data name="Buzzwords3" xml:space="preserve">
+    <value>ability;access;adapter;algorithm;alliance;analyzer;application;approach;architecture;archive;artificial intelligence;array;attitude;benchmark;budgetary management;capability;capacity;challenge;circuit;collaboration;complexity;concept;conglomeration;contingency;core;customer loyalty;database;data-warehouse;definition;emulation;encoding;encryption;extranet;firmware;flexibility;focus group;forecast;frame;framework;function;functionalities;Graphic Interface;groupware;Graphical User Interface;hardware;help-desk;hierarchy;hub;implementation;info-mediaries;infrastructure;initiative;installation;instruction set;interface;internet solution;intranet;knowledge user;knowledge base;local area network;leverage;matrices;matrix;methodology;middleware;migration;model;moderator;monitoring;moratorium;neural-net;open architecture;open system;orchestration;paradigm;parallelism;policy;portal;pricing structure;process improvement;product;productivity;project;projection;protocol;secured line;service-desk;software;solution;standardization;strategy;structure;success;superstructure;support;synergy;system engine;task-force;throughput;time-frame;toolset;utilisation;website;workforce</value>
+  </data>
+  <data name="Suffix" xml:space="preserve">
+    <value>s.r.l.;s.r.l.;s.r.l.;s.p.a.;s.p.a.;s.p.a.;s.c.a.r.l.;onlus</value>
+  </data>
+</root>
```

Comment:
With the exception of the Suffix value, this seem like a duplication of the english language.
If it is, could you please just remove them as there is no need to add them into a new resource file (ResourceManager should fallback to the default language (English) if no matching resource name was found